custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -129,8 +135,11 @@ def indexes(self, aggregation_level):
 
     def updates(self):
         yield """
-        DROP TABLE IF EXISTS "{temp_table}";
-        CREATE UNLOGGED TABLE "{temp_table}" AS
+        UPDATE "{tablename}" agg_awc SET
```

Comment:
nit: keeping this a temp_tablename would have been nice because the parameter passed to format is the temporary table. Alternatively could use f strings to reduce the extra varialbes being introduced